src/prop/prop_engine.cpp

```diff
@@ -313,6 +313,16 @@ bool PropEngine::isDecision(Node lit) const {
   return d_satSolver->isDecision(d_cnfStream->getLiteral(lit).getSatVariable());
 }
 
+std::vector<Node> PropEngine::getPropDecisions() const
+{
+  std::vector<Node> decisions; 
+  std::vector<SatLiteral> miniDecisions = d_satSolver->getDecisions();
+  for (SatLiteral d : miniDecisions) {
```

Comment:
Apply ClangFormat.

Reply:
Ran clang format and it did nothing.

Reply:
It should format the code according to our style guidelines. You can either follow https://github.com/cvc5/cvc5/wiki/Developer-Guide#formatting-with-clang-format, or simply install it with your package manager (e.g. `apt install clang-format` on ubuntu). To reformat all your changes relative to master, you can simply do `git clang-format master`.

In this case, it should only move the curly braces to the next line...

Reply:
> In this case, it should only move the curly braces to the next line...

Ah, I think it was a problem with my eyeballs and not clang format